File: KantechGatewayApp/Jobs/ProfileUpdateProcessor.vb
Namespace KantechGatewayApp.Jobs
    Public Class ProfileUpdateProcessor
        Inherits BaseProcessor
        Protected Overrides Sub HandleRow(row As Dictionary(Of String, String))
            Infrastructure.Logger.Info($"[ProfileUpdate] emp={row.GetValueOrDefault("employee ID")} dept={row.GetValueOrDefault("dept")} -> profile updated")
        End Sub
    End Class
End Namespace

File: KantechGatewayApp/Jobs/ProfileUpdateProcessor.cs
using System.Collections.Generic;

namespace KantechGatewayApp.KantechGatewayApp.Jobs
{
    public class ProfileUpdateProcessor : BaseProcessor
    {
        protected override void HandleRow(Dictionary<string, string> row)
        {
            Infrastructure.KantechJobApi.TriggerJobApi(_jobKey, row);
            Infrastructure.Logger.Info($"[ProfileUpdate] emp={row.GetValueOrDefault("employee ID")} dept={row.GetValueOrDefault("dept")} -> profile updated", _logSub);
        }
    }
}